docs/building-on-etherlink/bridging-wrapped-assets.mdx

```diff
@@ -26,38 +26,43 @@ It allows users to transfer several wrapped tokens, including:
-You should select the source chain from which you want to bridge a token into Etherlink. To send tokens out of Etherlink, select Etherlink as the source chain.
+1. Go to the Etherlink wrapped asset bridge at https://www.etherlinkbridge.com/bridge.
 
-### 3. Select the Token to Transfer
-Click **"Select Token"** to choose a token to transfer from the source chain, or alternatively, leave to the default source token. The interface will display all bridgeable tokens across all available networks as well as your token balances.
+1. Click the **Connect** button and connect your EVM wallet.
 
-### 4. Select the Amount of Tokens
-Enter the number of tokens you want to bridge from the source chain. The app will display:
-- The amount of bridged tokens you will receive on the destination chain
-- The necessary fees for the token transfer in the native gas token of the source chain. **Ensure you have enough gas on the source chain to pay the fees**
+1. Select the source network from which to bridge tokens to Etherlink, or select Etherlink as the source network to bridge tokens out of Etherlink.
 
- :::note Gas on destination
-Interacting with Etherlink requires XTZ (its native token) to pay for gas fees.
-As part of your bridging transaction, you can choose to send up to 0.1 XTZ to your Etherlink wallet address to pay the gas fees for later transactions with the Etherlink ecosystem.
-The cost of this XTZ is added to the fees that you pay for the transfer.
-By default, the bridge adds 0.1 XTZ if the destination account has no XTZ.
-:::
+1. Next to the source network, click the source token and select the token to transfer from the source network.
+The interface shows all bridgeable tokens across all available networks and your token balances.
 
-### 5. Transfer the Tokens
-- Click **"Transfer"**. Your wallet will ask you to approve the token spending and confirm the transaction
-- After your confirmation, a popup will appear; if you click on the link provided, you will be forwarded to [LayerZero Scan](https://layerzeroscan.com/) to observe the detailed progress of your bridge interaction
-- The bridge will also display the progress of your transaction, including an estimated period till the bridging is completed in the "transactions" box that appears after confirmation
-- Once the status changes to "completed", your tokens have arrived in your wallet
-- Please make sure you have imported the correct token into your wallet on the destination chain to see it
+1. Enter the number of tokens to bridge out of the source network.
+
+1. Select the target network and token.
+
+   The page shows the amount of tokens that you will receive on the target chain and the fees for the token transfer in the native gas token of the source chain.
+
+   **You must ensure that you have enough tokens on the source chain to pay the fees.**
+
+1. Set the other parameters for the transfer, including the **Gas on destination** and **Slippage tolerance** fields.
```

Comment:
Capitalize headings, so 'Gas on Destination' and 'Slippage Tolerance'

Reply:
They are lower case in the UI: 
<img width="510" alt="Screenshot 2024-07-30 at 11 38 01 AM" src="https://github.com/user-attachments/assets/5366a021-0bb8-42b5-8619-3e8e9aafacce">